runestone/common/project_template/_templates/plugin_layouts/sphinx_bootstrap/layout.html

```diff
@@ -114,11 +118,11 @@
           </a>
           <ul class='dropdown-menu'>
             {% if 'overview' in course_id %}
-                <li><a href='/{{appname}}/static/{{course_id}}/overview.html' aria-label="index-page">Table of Contents</a></li>
+                <li><a href='{{ pathto(master_doc) }}/overview.html' aria-label="index-page">Table of Contents</a></li>
             {% else: %}
                 <li><a href='{{ pathto(master_doc) }}' aria-label="index-page">Table of Contents</a></li>
             {% endif %}
-            <li><a href='/{{appname}}/static/{{course_id}}/genindex.html'>Book Index</a></li>
+            <li><a href='{{ pathto(master_doc) }}/genindex.html'>Book Index</a></li>
```

Comment:
This produces incorrect paths...

For example from index.html you get
```
<li><a href='#/genindex.html'>Book Index</a></li>
```


From another file I get:
```
<li><a href='index.html/genindex.html'>Book Index</a></li>
```

Reply:
I think we want to use `pathto("genindex.html", 1)` to give us a relative path to the file itself.

Reply:
Oops! Thanks, that's correct -- I've tested it on my install after a book rebuild.